src/Copy/Text.elm

```diff
@@ -432,52 +432,43 @@ t key =
 
         -- Get Help page
         GetHelpTitle ->
-            "[cCc] People You Can Talk To"
+            "People you can talk to"
```

Comment:
This copy wasn't locked yet, but I am happy for it to go in as long as we keep the copy card oen to verify all of it after today.